src/profile_manager/models.py

```diff
@@ -423,6 +428,32 @@ def post_delete_user(sender, instance, *args, **kwargs):
         instance.user.delete()
 
 
+@receiver(email_confirmed, sender=EmailConfirmationHMAC)
+def email_confirmed_handler(email_address, **kwargs):
+    """
+    After a user has confirmed their email, this will handle converting that email to the primary email
+    and therefore making it the User.email.
+
+    Then, we delete the non-primary email addresses of that user
+    """
+
+    with transaction.atomic():
+        email_address.set_as_primary()
+        EmailAddress.objects.filter(user=email_address.user, primary=False).delete()
+
+
+@receiver(user_logged_in, sender=User)
+def user_logged_in_verify_email_reminder_handler(request, user, **kwargs):
```

Comment:
I don't see a test for this, but I may have missed it.

Reply:
https://github.com/bytedeck/bytedeck/pull/1300/files#diff-d20ce8cc823f496ca1b2f26080ff2dc503de9450495c6efb9f35c79fb338babfR431

Reply:
Excellent.  make sense to have this covered in an integrated test.  Can you add a comment with the method name at this line, like:
# test of method: user_logged_in_verify_email_reminder_handler()

And if you are integrating tests of other methods, do the same thing?  Makes it easier to find with CTRL+F, and is one of the reason we use test naming convention:

`test_method_name__specific_details_being_tested`

Hmm... I guess that isn't documented anywhere.  I need to add that under contributing.